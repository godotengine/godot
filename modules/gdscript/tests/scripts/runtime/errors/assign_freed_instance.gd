class A extends Node:
	pass

func subtest_native():
	var x = Node.new()

	x.free()

	var _ok = x
	var _bad: Node = x

func subtest_script():
	var x = A.new()

	x.free()

	var _ok = x
	var _bad: A = x

func test():
	subtest_native()
	subtest_script()
