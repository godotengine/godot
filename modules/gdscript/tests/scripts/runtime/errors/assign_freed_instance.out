GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR at runtime/errors/assign_freed_instance.gd:10 on subtest_native(): Trying to assign invalid previously freed instance.
>> SCRIPT ERROR at runtime/errors/assign_freed_instance.gd:18 on subtest_script(): Trying to assign invalid previously freed instance.
